recipe/meta.yaml

```diff
@@ -144,7 +144,8 @@ outputs:
         - iminuit >=2
         - lmfit !=1.0.3
         - lsstdesc.coord
-        - matplotlib-base
+        # matplotlib-base 3.6 breaks fgcm; see DM-36267
+        - matplotlib-base <3.6
```

Comment:
This is fine, but I believe that the bug in matplotlib was fixed for 3.6.1.  Have you seen this as a problem now?  Or could we do `!=3.6.0`?

Reply:
I tested successfully with {{3.6.1}}

Reply:
Unpinned `matplotlib-base`